apps/wallet/src/ui/app/hooks/useGetCoinBalance.ts

```diff
@@ -16,6 +17,9 @@ export function useGetCoinBalance(
         {
             enabled: !!address && !!coinType,
             refetchInterval: 4000,
+            onError: (error) => {
```

Comment:
Do we want to show a toast here (basically same as https://mysten-labs.slack.com/archives/C042YF60NH2/p1677865742411389)?

Reply:
yes good catch